include/kafka/KafkaConsumer.h

```diff
@@ -716,17 +729,20 @@ KafkaConsumer::onRebalance(rd_kafka_resp_err_t err, rd_kafka_topic_partition_lis
         case RD_KAFKA_RESP_ERR__ASSIGN_PARTITIONS:
             KAFKA_API_DO_LOG(Log::Level::Info, "invoked re-balance callback for event[ASSIGN_PARTITIONS]. topic-partitions[%s]", tpsStr.c_str());
 
+            callRebalanceCallbackIfNecessary(err, tps);
```

Comment:
@niyue Thanks for your contribution!👍
Generally, the PR is fine. While some CI tests failed, since here we need to move `callRebalanceCallbackIfNecessary` a few lines down (after the line calling `_assign(tps);`.
The idea is, these rebalance callbacks should be called after partitions-assign, and before the partitions-revoke (otherwise some operations make no sense, right?).

Reply:
@kenneth-jia That makes sense and thanks for the guidance. I change calling the `callRebalanceCallbackIfNecessary` right after `_assign(tps);`, please check it out.

Reply:
Thanks, @niyue! Cheers!😊